bower.json

```diff
@@ -42,6 +42,9 @@
   "overrides": {
     "card": {
       "main": "lib/js/jquery.card.js"
+    },
+    "font-awesome": {
```

Comment:
ok, please check (if not done)  gulp serve:dist mode to ensure fontawesome icons are there in the "release mode"